Source/Eto.Wpf/Forms/ApplicationHandler.cs

```diff
@@ -76,7 +77,8 @@ void ApplyThemes()
 				return;
 
 			// Support (better) high DPI in the NumericUpDown control by theming the Extended WPF Toolkit's spinner.
-			Control.Resources.MergedDictionaries.Add(new sw.ResourceDictionary { Source = new Uri("pack://application:,,,/Eto.Wpf;component/themes/wpftoolkit/ButtonSpinner.xaml", UriKind.RelativeOrAbsolute) });
+			var assemblyName = Assembly.GetCallingAssembly ().GetName ().Name;
```

Comment:
Might want to use `typeof(ApplicationHandler).Assembly` instead, this will use the assembly that calls this method so it is indeterminate.